server/routerlicious/package.json

```diff
@@ -55,7 +56,7 @@
 		"stop": "docker compose down",
 		"stop:full": "docker compose down -v",
 		"test": "pnpm run -r --no-sort --stream --no-bail test",
-		"test:copyresults": "copyfiles --exclude \"**/node_modules/**\" \"**/nyc/**\" nyc",
+		"test:copyresults": "copyfiles --exclude \"**/node_modules/**\" \"packages/**/nyc/**\" nyc",
```

Comment:
CI failed locally with strange issues here, and when I was debugging `copyresults` I realized that the input glob here can match files written to output. Not sure exactly how that manifests in CI not finding results files, but given copyfiles looks to be highly parallelized, it seemed worth a shot to just make this change.

Reply:
This didn't fix the issue, but on further investigation I also noticed that we were reporting coverage on everything in the server directory including node_module dependencies. Adding some reasonable configurations that resemble the package root seems to have fixed this in latest build (hopefully it's not a flaky thing that reoccurs).

Hitting a limit like this while changing deps makes some vague amount of sense; I believe the newer version of flub has a heavier footprint. I do know at some point I fixed an issue with coverage caused by OOM issues merging reports when not using the merge-async option, so it seems plausible to me this pushed something in the report writing over the edge on CI machines.

Reply:
Any ideas why there would be any nyc directories (or src/lib/dist ones) outside packages/ ? 🤔

Reply:
It's specifically the one we write here--this copyfiles command takes all the per-package nyc results and sticks them in the root nyc folder.

This didn't end up being the actual problem, but only matching per-package `nyc` folders for copying seems like a good move anyway so I kept the change.